README.md

```diff
@@ -141,36 +154,41 @@ yarn start # start the dev server
 
 A development server should now be running on [localhost:4000](localhost:4000).
 
-## Step 2. View your Blocks within a sandbox
+## Step 2. View your development server within the Blocks app
 
-When you visit [localhost:4000](localhost:4000) in your browser, you'll see an interface where you can test out your Block.
+When you visit [localhost:4000](localhost:4000) in your browser, you'll be
+redirected to the Blocks app with your locally-developed blocks embedded in it.
+You can see your blocks by clicking on the block picker toward the top of the
+page (they're shown at the top in blue with a plug icon):
 
-This starter project has one example folder block and one example file block.
+<img src="https://user-images.githubusercontent.com/56439/181648955-101b6567-3f9b-44b3-af99-7ef3ca6161b9.png" width="418" />
 
-[Sandbox video](https://user-images.githubusercontent.com/8978670/144666304-b9012177-aed6-4c26-afc3-3de8c8d6d0ad.mov)
-
-You can play with two bits of the interface to view your Blocks:
-
-1. **Input file or folder**: An input that accepts the path to a file or a folder on github.com. Here are some valid file/folder paths.
+This starter project has one example folder block and one example file block.
+Try choosing one of the example blocks in the block picker to see it in action.
+When you make changes to the block code it will be hot-reloaded in the app.
 
-- https://github.com/facebook/react/blob/main/README.md (file)
-- https://github.com/facebook/react/blob/0.3-stable/README.md (file)
-- https://github.com/facebook/react/blob/main/packages/react-dom/index.js (file)
-- https://github.com/facebook/react/ (folder)
-- https://github.com/facebook/react/tree/0.3-stable (folder)
-- https://github.com/facebook/react/tree/main/packages (folder)
+If you don't see your blocks, make sure that:
 
-2. **List of custom blocks**: A menu of the blocks that you have defined in `blocks.config.json` (see below; for now this contains the example file and folder blocks)
+- your repo is public
+- your development server is running
+- there's a `devServer` query parameter in the URL pointing to your development server
 
-Once you've entered a valid path, choose a block from the menu; the file or folder content should be rendered beneath. Check your console for errors if something isn't working!
+If you prefer not the make your blocks development repo public, it is still
```

Comment:
I _think_ by "blocks development repo public" we're referring to the repo with the contents the Block is viewing, right? Might be worth separating that from the repo where you're developing the Block

Reply:
here we are talking about the repo where you're developing the block. This is confusing and tripped me up several times: it is necessary either for the blocks development repo to be public or for the GitHub App to be installed on it, since we call `/repos/{owner}/{repo}` on it.

The reason we do that call is only to get the repo ID, which we use for Cosmos DB. This is pretty annoying! We could just drop it and have Cosmos DB calls fail in dev mode, kind of a bummer but would remove a broken stair from the common-case dev experience.

Or we could fix the Cosmos DB code somehow so it doesn't need the id—we use the block owner/repo/id as the table name, and we get only 63 characters, so I decided to use the repo id instead of the string owner/repo. But since we are the only users it would probably be ok to use `githubnext-blocks-examples-emoji-block' (39 characters).

Oh and now I remember the other issue, the available characters for table names are `[A-Za-z0-9-]`, so there is no good way to delimit the owner / repo — someone could access the emoji block data (🙀 ) by making an owner `githubnext-blocks` and repo `examples`. This is probably not worth worrying about at this point!

I am inclined to change the Cosmos DB table names to clear this up, what do you think?

Reply:
removed the paragraph in anticipation of changing the Cosmos DB table names